lib/ACF/AlterComponents.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ *  Copyright (c) 2021. Geniem Oy
+ */
+
+namespace TMS\Theme\Amuri\ACF;
+
+use TMS\Theme\Amuri\ACF\Layouts\CustomHtmlLayout;
+
+/**
+ * AlterComponents
+ */
+class AlterComponents {
+
+    /**
+     * Constructor
+     */
+    public function __construct() {
+        add_filter(
```

Comment:
Namespacen ulkoiset hyvä kenottaa
esim. add_filter(), apply_filters()
```suggestion
        \add_filter(
```